src/core/ext/filters/client_channel/subchannel_stream_client.cc

```diff
@@ -250,10 +250,10 @@ void SubchannelStreamClient::CallState::StartCallLocked() {
   payload_.send_initial_metadata.peer_string = nullptr;
   batch_.send_initial_metadata = true;
   // Add send_message op.
-  send_message_.emplace();
-  send_message_->Append(Slice(
+  send_message_.Clear();
```

Comment:
I think this isn't necessary, since we're only ever sending a single message, so the slice buffer should always be empty at this point.